### PR TITLE
Made predictions for mobile price classification

### PR DESCRIPTION
**Pull request by Rohith04MVK**
Made predictions for mobile price classification with 95% accuracy.

### CODE REVIEW

File: src/mobile-price-classification/main.py [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+
+train_df = pd.read_csv("train.csv")
+
+X = train_df.drop(["price_range"], axis=1).values
+y = train_df["price_range"].values
+
+# Feature selection
+fs = SelectKBest(score_func=f_classif, k=10)
+X = fs.fit_transform(X, y)
+print(f"Shape of X = {X.shape}")
```
Comment (janaSunrise) [inline src/mobile-price-classification/main.py:16]:
```suggestion
```
No debug code ;)

**Review by janaSunrise — CHANGES_REQUESTED** [review]

**Review by janaSunrise — APPROVED** [review]
Awesome! good to go 👍🏼
